### ISSUE TITLE
GoAccess errors when a valid log contains only entries from excluded IPs

### ISSUE THREAD

**Issue by organman91** [allinurl/goaccess#519]
Using version 1.0.2:

If I pass in a log file containing only entries that are later filtered via --exclude-ip, goaccess will exit with an error message, instead of generating an empty report:

```
GoAccess - version 1.0.2 - Jul  4 2016 19:44:08
Config file: /etc/goaccess.conf

Fatal error has occurred
Error occured at: src/goaccess.c - main - 1230
Nothing valid to process. Verify your date/time/log format.
```

My team runs goaccess in a cron job, so we'd like a way to ensure that goaccess exits cleanly (exit code 0, no output to stdout but only to report file).

**Comment by allinurl** [comment]
Thanks for pointing this out. It will be addressed on issue #471.

**Comment by allinurl** [comment]
This has been implemented. I added the ability to show which token didn't match log/date/time pattern.

This also ensures that in the absence of data, its output is not  treated as error but instead it produces an empty report.

Feel free to build from [**development**](https://goaccess.io/download#build) to test it. It will be pushed out in the upcoming release.
